install/aws/dev/docker/php-fpm/Dockerfile

```diff
@@ -34,74 +32,65 @@ RUN apk add --update --no-cache \
     libjpeg-turbo-dev \
     freetype-dev \
     imagemagick \
-    nodejs \
-    npm \
     git \
     html2text \
-    nginx \
     libmemcached-libs \
     zlib \
     postgresql-dev \
     tidyhtml-dev \
-    gettext-dev
+    gettext-dev && \
 
-# Memcached PHP lib
-RUN /src/setup-memcached.sh
-
-# TODO - let's see if we actually need xdiff and if so can we include it as an artifact or docker layer
+    # Memcached PHP lib
+    /src/setup-memcached.sh
 
 # Install the Wikijump repository
 WORKDIR /var/www
-RUN git clone \
-    --depth 10 \
-    --branch "${WIKIJUMP_REPO_BRANCH}" \
-    "${WIKIJUMP_REPO}" "${WIKIJUMP_REPO_DIR}"
+
+    git clone \
```

Comment:
No `RUN`? This isn't a continuation of a previous command, this seems like an error.

Reply:
You're right, the `RUN` statement was on a local-only line, thanks.